kiwi/repository/dnf.py

```diff
@@ -253,9 +263,9 @@ def _create_runtime_config_parser(self):
         self.runtime_dnf_config.set(
             'main', 'gpgcheck', self.gpg_check
         )
-        self.runtime_dnf_config.set(
-            'main', 'repo_gpgcheck', self.gpg_check
-        )
+        # We are not setting repo_gpgcheck, which forces repository
+        # signatures checks, because current Fedora 25 releases do not
+        # provide signed repositories, only signed packages
```

Comment:
It is possible for distributions to have signed repositories, or even individual repos that do. For example, in Fedora, the `fedora-cisco-openh264` repo is signed.